Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM ubuntu:18.04 as builder
 LABEL maintainer="info@camptocamp.com"
 
 RUN apt-get update && \
+    apt-get update --yes && \
```

Comment:
this is almost the same line as above, can't this be merged?

Reply:
You say it, it's almost the same line, not the same ...

Reply:
:+1:

Reply:
Ok but I  still do not understand why we run `apt update` without `--yes` option.

Reply:
update: update the package dist with version, ...
upgrade: upgrade the installed package to the latest version

Reply:
Ok, so the second one should be `upgrade` instead of `update` as I understand.

Reply:
Done :-)